modules/virt-importing-vm-cli.adoc

```diff
@@ -5,7 +5,7 @@
 [id="virt-importing-vm-cli_{context}"]
 = Importing a virtual machine with the CLI
 
-You can import a virtual machine with the CLI by creating the Secret and VirtualMachineImport Custom Resources (CRs). The Secret CR stores the RHV Manager credentials and CA certificate. The VirtualMachineImport CR defines the parameters of the VM import process.
+You can import a virtual machine with the CLI by creating the secret and virtual machine import Custom Resources (CRs). The secret CR stores the RHV Manager credentials and CA certificate. The virtual machine import CR defines the parameters of the VM import process.
```

Comment:
Again, since these are described as custom resources, they should get the backticks/pascalcase ("custom resources" should not be capitalized, though)